recipes/fabio/meta.yaml

```diff
@@ -36,6 +36,8 @@ about:
     home: https://github.com/kif/fabio
     license: GPL v3.0, GPL v2.0, MIT, GPL, LGPL v3.0, BSD 3-Clause, Apache 2.0
     summary: I/O library for images produced by 2D X-ray detector
+    license_family: Other
+    license_file: copyright
```

Comment:
You need to commit the `copyright` file too. (Or copy it from the `SRC_DIR` to the recipe dir, never tried that though.)